chainmanager/types/params.go

```diff
@@ -144,5 +150,10 @@ func ParamKeyTable() subspace.KeyTable {
 func DefaultParams() Params {
 	return Params{
 		TxConfirmationTime: DefaultTxConfirmationTime,
+		ChainParams: ChainParams{
+			BorChainID:           helper.DefaultBorChainID,
+			StateReceiverAddress: DefaultStateReceiverAddress,
+			ValidatorSetAddress:  DefaultValidatorSetAddress,
+		},
```

Comment:
Other contract addresses will have empty string ("") instead of ("000000000000000000000000000000000000000") in genesis.json. is that okay??

Reply:
Fixed.